editor/gui/scene_tree_editor.cpp

```diff
@@ -399,11 +399,16 @@ void SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent) {
 		if (!scr.is_null()) {
 			String additional_notes;
 			Color button_color = Color(1, 1, 1);
+			StringName class_name = p_node->get_class_name();
 			// Can't set tooltip after adding button, need to do it before.
 			if (scr->is_tool()) {
 				additional_notes += "\n" + TTR("This script is currently running in the editor.");
 				button_color = get_theme_color(SNAME("accent_color"), EditorStringName(Editor));
 			}
+			if (class_name != scr->get_instance_base_type()) {
+				additional_notes += "\n" + TTR("Script does not inherit native type " + class_name);
```

Comment:
```suggestion
				additional_notes += "\n" + TTR("Script does not inherit native type") + " " + class_name;
```
You can't put dynamic strings in `TTR` it will break all the translations

Reply:
Actually that wasn't a correct suggestion. It's true that strings shouldn't be concatenated like this within `TTR()`, but translations shouldn't be concatenated at all - different languages have different sentence structures and need to play words in different places.

So this needs to use `vformat` with the `%s` substitution marker so that it can be translated properly:
```cpp
TTR(vformat("Script does not inherit native type \"%s\".", class_name))
```
(added quotes while at it to make it clear that `%s` is the "native type", which will also help keep translations clear)

Reply:
That is true, wasn't sure about which to use here, thank you

Reply:
I'll add this suggestion and squash another commit in a minute.

Just waiting for my local action jobs to finish.